packages/core/src/lib/createSystem.ts

```diff
@@ -107,15 +107,19 @@ export function createSystem(config: KeystoneConfig) {
     graphQLSchema,
     adminMeta,
     getKeystone: (prismaModule: PrismaModule) => {
```

Comment:
This is great to see @mmachatschek thanks!!

I have been playing around with the new Prisma Planetscale Serverless driver with Keystone `getContext`. I have got it working here https://github.com/borisno2/on-the-hill-drama-club/pull/189 I wonder is we can also make this usecase easier or less confusing as to which "Prisma Client" would actually be used in that instance. I wonder if we can work towards maybe `prismaModule` optional here in `getKeystone` and by extension `getContext`? 

From a DX perspective it feels strange to pass in the Prisma Client twice, and I also found it unnecessarily complex to pass in an customised Prisma Client into `getContext`.

Reply:
Oh that's a good point, `getContext` requiring a `PrismaModule` is not reasonable with this change

Reply:
@borisno2 @dcousens  I pushed a commit to attempt to remove passing in the prismaModule into the `getKeystone` and `getContext` functions

https://github.com/keystonejs/keystone/pull/8847/commits/9fe430c2a8fbfc308b8860352684d4ca733685db

We now accept a `prismaClientModule` function that returns a `PrismaClient` instance and the `Prisma` object. 
If that function (`prismaClientModule`) exists, we can assume that the user passes in a prisma client and don't need to instantiate and import it anymore.